client/gui/CGuiHandler.h

```diff
@@ -62,6 +64,7 @@ class CGuiHandler
 	               textInterested;
 
 
+	void handleMouseButtonClick(CIntObjectList &interestedObjs, EIntObjMouseBtnType btn, bool isPressed);

```

Comment:
Yeaaah, sorry. I was trying to set this in visual studio formatter and then kinda forgot (and, apparently, visual studio doesn't have the option to change that...). Will fix.

I'm not really up-to-date with c development -- isn't there some portable way for a project to have correct code style used by IDEs by default? Clang format?